src/public/index.html

```diff
@@ -21,67 +29,79 @@ <h1 class="red-border-bottom">Join Us</h1>
-            <p>Fill out form below to stay connected to this campaign. We will only contact you when there is something you can do.</p>
-            <form action="submit" method="post">
-                <div class="form-group">
-                    <label for="name">Name</label>
-                    <input class="form-control input-lg" type="text" name="name" id="name" required>
-                </div>
-                <div class="form-group">
-                    <label for="email">Email address</label>
-                    <input class="form-control input-lg" type="email" name="email" id="email" required>
-                </div>
-                <div class="form-group">
-                    <label for="phone">Phone number</label>
-                    <input class="form-control input-lg" type="phone" name="phone" id="phone">
-                </div>
-                <input type="hidden" name="campaign" id="campaign" value="some_campaign">
-                <div class="form-group">
-                    <button type="submit" class="btn btn-primary btn-lg">Get Involved</button>
-                </div>
-            </form>
+          <p>Fill out form below to stay connected to this campaign. We will only contact you when there is something you can do.</p>
+          <form action="submit" method="post">
+            <div class="form-group">
+              <label for="name">Name</label>
+              <input class="form-control input-lg" type="text" name="name" id="name" required>
+            </div>
+            <div class="form-group">
+              <label for="email">Email address</label>
+              <input class="form-control input-lg" type="email" name="email" id="email" required>
+            </div>
+            <div class="form-group">
+              <label for="phone">Phone number</label>
+              <input class="form-control input-lg" type="phone" name="phone" id="phone">
+            </div>
+            <div class="form-group">
+              <label for="zip">Zip code</label>
+              <input class="form-control input-lg" type="zip" name="zip" id="zip" required pattern="[0-9]{5}" title="Five-digit U.S. zip code">
+            </div>
+            <input type="hidden" name="campaign" id="campaign" value="inauguration">
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-lg">Get Involved</button>
+            </div>
+          </form>
         </section>
       </div>
 
       <footer>
-          <p>Other links here?</p>
+        <p>Other links here?</p>
       </footer>
-    </body>
+    </div>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.6/handlebars.min.js" integrity="sha256-1O3BtOwnPyyRzOszK6P+gqaRoXHV6JXj8HkjZmPYhCI=" crossorigin="anonymous"></script>
```

Comment:
loading handlerbars from cdn instead of locally. I also switched to `cdnjs.com` cuz they have a lot more libs available and PageSpeedTest recommends using a single cdn if possible. I also removed the bootstrap JS cuz we're not using that currently afaict.